src/components/DetailsMetrics/DetailsMetrics.js

```diff
@@ -181,24 +315,106 @@ const DetailsMetrics = ({ selectedItem }) => {
   }
 
   return (
-    <>
+    <div className="metrics">
       {generatedMetrics.map(([applicationName, applicationMetrics]) => {
         return (
-          <div key={applicationName} className="metrics">
-            <div className="metrics__app-name">{applicationName}</div>
+          <React.Fragment key={applicationName}>
+            <div className="metrics__app-name">
+              {applicationName === mlrunInfra ? '' : applicationName}
+            </div>
             {applicationMetrics.map(metric => {
-              if (!metric.data) {
+              if (applicationName === mlrunInfra) {
+                if (!metric.data)
+                  return (
+                    <NoMetricData
+                      className="empty-invocation-card-sticky"
```

Comment:
no need to specify the state `sticky` in the className

Reply:
done.